doc/man/scons.xml

```diff
@@ -3310,36 +3317,48 @@ arguments
 to avoid unnecessary test execution.</para>
   </listitem>
   </varlistentry>
-</variablelist>
 
-<para>A created
-<emphasis role="bold">Configure</emphasis>
-instance has the following associated methods:</para>
-
-<variablelist>
   <varlistentry>
   <term>SConf.Finish(<emphasis>context</emphasis>)</term>
-  <term><emphasis>sconf</emphasis>.Finish()</term>
-  <listitem>
-<para>This method should be called after configuration is done.
-It returns the environment as modified
-by the configuration checks performed.
+  <term><replaceable>context</replaceable>.Finish()</term>
+  <listitem>
+<para>This method must be called after configuration is done.
+Though required, this is not enforced except
+if &Configure; is called when there still an active context,
+in which case an exception is raised.
+&Finish; returns the environment as modified
+during the course of running the configuration checks.
 After this method is called, no further checks can be performed
 with this configuration context.
 However, you can create a new
-Configure
-context to perform additional checks.
-Only one context should be active at a time.</para>
-
-<para>The following Checks are predefined.
-(This list will likely grow larger as time
-goes by and developers contribute new useful tests.)</para>
+configure context to perform additional checks.
+</para>
   </listitem>
   </varlistentry>
+</variablelist>
+
+<para>Example of a typical Configure usage:</para>
```

Comment:
this was just dangling in the middle of the list of methods, moved up here to give a starter idea of what it works like.